src/main/java/Hari.java

```diff
@@ -1,4 +1,9 @@
 import java.util.Scanner;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.FileReader;
+import java.io.BufferedReader;
+import java.io.IOException;
 
 // Class for the chatbot itself
 class handlerbot {
```

Comment:
Perhaps renaming handlerbot to HandlerBot to follow PascalCase convention?

Reply:
Thanks! Have made the necessary amendments for this and the others you highlighted